routesrv/eskipbytes.go

```diff
@@ -15,6 +19,7 @@ import (
 // serve as an HTTP handler exposing its content.
 type eskipBytes struct {
 	data        []byte
+	etag        string
```

Comment:
Lets keep track of modification timestamp and pass it to `http.ServeContent` - it would set `Last-Modified` header which is handy.

Reply:
Good suggestion. 
**Fun fact**: Did you know `If-Modified-Since`  and `If-Unmodified-Since`  is used for the same purpose (makes requests conditional) but unlike `If-Unmodified-Since`, `If-Modified-Since` can only be used with a `GET`  or a `HEAD`